### ISSUE TITLE
docker-compose.yml process exits prematurely with errors

### ISSUE THREAD

**Issue by ojonesjr** [mautic/docker-mautic#136]
When trying to run the following, taken from your example, the process fails and all containers exit. This same issue happens both on https://labs.play-with-docker.com/ and on my personal docker setup. This is the example code that you have:

```
version: "2"
services:
  mautic:
    restart: unless-stopped
    image: mautic/mautic
    depends_on:
      - mysql
    ports:
      - "80:80"
    environment:
      MAUTIC_DB_HOST: mysql
      MAUTIC_DB_USER: mautic
      MAUTIC_DB_PASSWORD: mauticdbpass
      MAUTIC_TRUSTED_PROXIES: 0.0.0.0/0
    volumes:
      - mautic-web:/var/www/html

  mysql:
    restart: unless-stopped
    image: mysql:5.6
    environment:
      MYSQL_ROOT_PASSWORD: mysqlrootpassword
      MYSQL_DATABASE: mautic
      MYSQL_USER: mautic
      MYSQL_PASSWORD: mauticdbpass
    volumes:
      - mysql:/var/lib/mysql
```

And this it the errors that I get:

```
Could not write configuration file to /var/www/html/app/config/local.php, you can create this file with the following contents:

<?php
$parameters = array (
  'db_driver' => 'pdo_mysql',
  'install_source' => 'Docker',
  'db_host' => 'mysql',
  'db_name' => 'mautic',
  'db_user' => 'mautic',
  'db_password' => 'mauticdbpass',
  'trusted_proxies' => 
  array (
    0 => '0.0.0.0/0',
  ),
  'default_timezone' => 'UTC',
);
```

chown: cannot access 'app/config/local.php': No such file or directory

**Any advise would be sincerely appreciated.**

**Comment by sr-verde** [comment]
Have a lookt at: #135

**Comment by ojonesjr** [comment]
Same issue being resolved in #135
